Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileItem.cs

```diff
@@ -1,42 +1,41 @@
 ﻿using Renci.SshNet.Sftp;
 
-namespace Frends.SFTP.DownloadFiles.Definitions
-{
-    internal class FileItem
-    {
-        public DateTime Modified { get; set; }
+namespace Frends.SFTP.DownloadFiles.Definitions;
 
-        public string Name { get; set; }
+internal class FileItem
+{
+    public DateTime Modified { get; set; }
 
-        public string NameWithMacrosExtended { get; set; }
+    public string Name { get; set; }
 
-        public string FullPath { get; set; }
+    public string NameWithMacrosExtended { get; set; }
 
-        public long Size { get; set; }
+    public string FullPath { get; set; }
 
-        public FileItem(SftpFile file)
-        {
-            Modified = file.LastWriteTime;
-            Name = file.Name;
-            Size = file.Length;
-            FullPath = file.FullName;
-        }
+    public long Size { get; set; }
 
-        public FileItem(string fullPath)
-        {
-            if (!File.Exists(fullPath))
-                throw new ArgumentException($"File does not exist: '{fullPath}");
+    public FileItem(SftpFile file)
+    {
+        Modified = file.LastWriteTime;
+        Name = file.Name;
+        Size = file.Length;
+        FullPath = file.FullName;
+    }
 
-            var fi = new FileInfo(fullPath);
-            Modified = fi.LastWriteTime;
-            Name = Path.GetFileName(fullPath);
-            Size = fi.Length;
-            FullPath = fullPath;
-        }
+    public FileItem(string fullPath)
+    {
+        if (!File.Exists(fullPath)) throw new ArgumentException($"File does not exist: '{fullPath}");
 
-        /// <summary>
-        /// Default constructor, use only for testing.
-        /// </summary>
-        public FileItem() { }
+        var fi = new FileInfo(fullPath);
+        Modified = fi.LastWriteTime;
+        Name = Path.GetFileName(fullPath);
+        Size = fi.Length;
+        FullPath = fullPath;
     }
+
+    /// <summary>
+    /// Default constructor, use only for testing.
+    /// </summary>
+    public FileItem() { }
```

Comment:
You are not using this constructor anywhere, so it can be removed.

Reply:
Removed.